src/main/kotlin/io/github/mojira/arisa/infrastructure/jira/Functions.kt

```diff
@@ -35,6 +36,46 @@ fun connectToJira(username: String, password: String, url: String): Pair<JiraCli
     return JiraClient(url, credentials) to credentials
 }
 
+/**
+ * Returns a list of strings indicating ticket IDs that are contained in the given jql filter.
+ * TODO: Actually return the tickets themselves instead of only ticket IDs.
+ *
+ * @param cap - How many tickets should be queried at most. No cap by default.
+ */
+fun searchIssues(jiraClient: JiraClient, jql: String, cap: Int = Int.MAX_VALUE) = runBlocking {
```

Comment:
You are not using this cap you are still using MAX_RESULTS 👀

Reply:
I'm using the cap in L68

Reply:
you can put it instead of MAX_RESULTS and then we dont query as much data from Jira :D (I think its set to 200), 

That also means you dont need to check on each loop

Reply:
Yep, true. I added the cap after the fact just as a safety measure. Would be better to move it directly to replace `MAX_RESULTS` -- then we can also get rid of the loop altogether.